tests/test_config_section_global.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+"""
+Test validating of GlobalSection.
+"""
+
+from base import TestBase
+
+from virtwho.config import GlobalSection, str_to_bool, MinimumSendInterval, DefaultInterval
+from virtwho.log import DEFAULT_LOG_DIR
+
+# Values used for testing GlobalConfigSection
+GLOBAL_SECTION_VALUES = {
+    'debug': True,
+    'oneshot': 'true',
+    'print': 'false',
+    'wrong_bool_value': 'nein',
+    'configs': ['local_libvirt.conf'],
+    'log_file': 'my_custom.log',
+    'interval': '120'
+}
+
+
+class TestGlobalConfigSection(TestBase):
+    """
+    Test base for testing class GlobalSection
+    """
+
+    def init_global_config_section(self):
+        """
+        Method executed before each unit test
+        """
+        self.global_config = GlobalSection('global', None)
+        # We need to set values using this way, because we need
+        # to trigger __setitem__ of global_config
+        for key, value in GLOBAL_SECTION_VALUES.items():
+            self.global_config[key] = value
+
+    def test_validate_debug_value_bool(self):
+        """
+        Test validating of correct bool value 
+        """
+        self.init_global_config_section()
+        result = self.global_config._validate_str_to_bool('debug')
+        self.assertIsNone(result)
+        value = self.global_config.get('debug')
+        self.assertEqual(value, True)
```

Comment:
Just a thought: Depending on what you want, you may what to assert identity here instead of equality.  E.g.

```
In [1]: 1 == True
Out[1]: True

In [2]: 1 is True
Out[2]: False
```